components/raftstore-v2/src/batch/store.rs

```diff
@@ -64,19 +64,17 @@ struct StorePoller<EK: KvEngine, T> {
 
 impl<EK: KvEngine, T> StorePoller<EK, T> {
     pub fn new(poll_ctx: StoreContext<T>, cfg_tracker: Tracker<Config>) -> Self {
-        let mut poller = Self {
+        Self {
             store_msg_buf: Vec::new(),
             peer_msg_buf: Vec::new(),
             poll_ctx,
             cfg_tracker,
             last_flush_time: TiInstant::now(),
             need_flush_events: false,
-        };
-        poller.apply_buf_capacity();
```

Comment:
Receiving messages depends on the capacity. Without initializing the capacity, it will never receive any messages.

Reply:
I deleted it because of your https://github.com/tikv/tikv/pull/13127#discussion_r930583660. But yes, apparently I wasn't paying attention either.

Reply:
I have another PR fixing it by passing an argument explicitly. Using capacity is too obscure.